fbrp/setup.py

```diff
@@ -13,9 +13,10 @@
     ),
     python_requires=">=3.7",
     install_requires=[
-        "aiodocker",
-        "docker",
-        "six",
-        "alephzero==v0.3",
+        "aiodocker==0.21.0",
+        "alephzero>=v0.3",
+        "docker==5.0.0",
+        "psutil==5.8.0",
+        "six==1.16.0",
```

Comment:
Would this be more robust if we used `>=` every where instead of `==` ?

Reply:
Done.